Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -142,8 +142,9 @@ class Detector : public AliceO2::Base::Detector
                                     UInt_t &dettype) const;
 
     /// This method is an example of how to add your own point of type Point to the clones array
-    AliceO2::ITSMFT::Point *addHit(Int_t trackID, Int_t detID, TVector3 startPos, TVector3 pos, TVector3 mom, Double_t startTime,
-                  Double_t time, Double_t length, Double_t eLoss, Int_t shunt, Int_t status, Int_t statusStart);
+    AliceO2::ITSMFT::Point *addHit(int trackID, int detID, TVector3 startPos, TVector3 endPos, TVector3 startMom,
```

Comment:
@sawenzel : as far as I know, the momentum is needed for the hit -> parameterized digits generation, I think it was put there by @iouribelikov for this purpose (in my original aliroot version there was no momentum). Yura, please confirm.

The TVector3 is used because currently the particle position and momentum is extracted from geant via TLorentzVector. So, now we use min.overhead approach to use
    inline TVector3 TLorentzVector::Vect() const { return fP; }
see  Detectors/ITSMFT/ITS/simulation/src/Detector.cxx:426

If there will be better interface to geant we can change this. Internally hit keeps TVector3<float>